### PR TITLE
Issues/#1580 vocab dcat2

### PR DESCRIPTION
**Pull request by barthanssens**
GitHub issue resolved: #1580  

Briefly describe the changes proposed in this PR:

* Added DCAT 2 classes and properties (same namespace as DCAT)
* Added TIME, PROV-O, ODRL ontologies referenced in DCAT2

### CODE REVIEW

File: core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/TIME.java [hunk 1]
```diff
@@ -0,0 +1,419 @@
+/**
```
Comment (jeenbroekstra) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/TIME.java]:
```suggestion
/*******************************************************************************
```

File: core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/TIME.java [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
```
Comment (jeenbroekstra) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/TIME.java]:
```suggestion
 *******************************************************************************/
```

**Comment by jeenbroekstra** [comment]
Also: can you squash before you merge?

**Comment by barthanssens** [comment]
OK, will do, I'm still waiting a bit because the spec is in public review (Candidate Recommendation until at least Oct. 31st)

**Pull request merged**